src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1192,13 +1192,15 @@ is_element_type_primitive (MonoType *vector_type)
 static MonoInst*
 emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsig, MonoInst **args)
 {	
+#if defined(TARGET_AMD64) || defined(TARGET_WASM)
 	if (!COMPILE_LLVM (cfg))
 		return NULL;
-
-#ifdef TARGET_ARM64
-	if (!(cfg->compile_aot && cfg->full_aot && !cfg->interp))
-		return NULL;
 #endif
+// FIXME: This limitation could be removed once everything here are supported by mini JIT on arm64
+// #ifdef TARGET_ARM64
+// 	if (!(cfg->compile_aot && cfg->full_aot && !cfg->interp))
+// 		return NULL;
+// #endif
```

Comment:
Temporary commented these lines out for testing on CI. Need to uncomment before merging.